.github/workflows/proto.yml

```diff
@@ -19,6 +19,7 @@ jobs:
     steps:
       - uses: actions/checkout@v3
       - name: "Check protobuf generated code matches committed code"
+        # yamllint disable
```

Comment:
[Non-Blocking][Suggestion]
Instead of disabling yaml lint, probably fix the indentation like mentionned here: https://stackoverflow.com/a/69825610/11529598

Reply:
That is a different issue, that stackoverflow is asking about invalid workflow, which is Github linting the workflow, not `yamllint`.

Yamllint was failing for line length. I choose to disable the lint for these blocks because I didn't want to mess with the code block for something basic like line length.

Reply:
same response.